pex/pep425.py

```diff
@@ -99,6 +99,14 @@ def _iter_supported_tags(cls, impl, version, platform):
     """
     # Predict soabi for reasonable interpreters.  This is technically wrong but essentially right.
     abis = []
+    if impl == 'cp' and version.startswith('2'):
```

Comment:
I think it's safe to assume based on pip's implementation that this initial set will apply to both cp2 and cp3. if other python3 specific items beyond 'abi3' are needed, presumably they'd be additive and could be handled in the `if version.startswith('3')` branch.